### ISSUE TITLE
Support Emacs 24 Themes

### ISSUE THREAD

**Issue by ajsquared** [ajsquared/color-theme-x#1]
Emacs 24 themes are not supported by this package.  Add support for these themes in addition to the older style.

**Comment by mattiasb** [comment]
Was this ever fixed?

**Comment by ajsquared** [comment]
Hmm, I don't think this was actually fixed properly.

I probably won't be able to look into this for a little while, but if you're interested I'm always happy to get contributions.

**Comment by mattiasb** [comment]
Nah, I was just interested in testing this out. I haven't used any of the old pre-24 themes so I don't know where to get one. My plan was to just test this out on wombat (the theme I'm using) and it didn't work. Hence the question. :)

**Comment by ajsquared** [comment]
No worries, thanks for bringing this to my attention
